plugin-api.d.ts

```diff
@@ -450,6 +450,8 @@ interface VectorSegment {
 interface VectorRegion {
   readonly windingRule: WindingRule
   readonly loops: ReadonlyArray<ReadonlyArray<number>>
+  readonly fills?: ReadonlyArray<Paint>
+  readonly fillStyleId?: { key: string, version: string } | null
```

Comment:
Can we use the same format for `region.fillStyleId` as we use for `node.fillStyleId`? I.e. just a plain string? Otherwise users will have to encode/decode the ids when using this in combination with `style.id` and `figma.getStyleById`